src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -168,15 +187,15 @@ contract SecurityCouncilManager is Initializable, ArbitrumTimelock, ISecurityCou
         onlyRole(MEMBER_ROTATOR_ROLE)
```

Comment:
- the idea of the sig is to ensure that the rotateMember path doesn't grant the SC any additional affordances; i.e., 9 of the 12 can't rotate out any of the other 3 without their consent.
- agreed re replaceMember would be useful anyway (since if the SC is removing, they'll practically speaking probably already have a replacement   
- i'm thinking a refactor that guts out the add/remove method internals into _addMemberToCohortArray and _removeMemberFromCohortArray, so that replaceMember and rotateMember can reuse them like you suggested without two _schedule()s being called

Reply:
pushed, lmk wu think

Reply:
Looks good, although I'm still not fully convinced on the rotate member sig verification. Lets chat more about it.

My thoughts are: Technically the sec council can do it wants by just calling the replaceMember func. Or by upgrading to add whatever they like. So in some senses this just to more clearly dilleanate the boundry, but I think forcibly key rotating a member without their permission already pretty obviously bad - activity like this would have to be pre-announced anyway to avoid panic. Additionally, one reason for rotation is if a key has been lost, in this case the member may not even be able to sign for the rotation.

Reply:
the possibility of keys being lost is reason enough to convince me that there shouldn't be sig verification. i think there should be separate methods for remove vs rotate, so there's on-chain code-path distinction in the security council's intent (new address vs. whole new meat-space identity)